jenkins/job-configs/kubernetes-jenkins/kubernetes-upgrade.yaml

```diff
@@ -0,0 +1,296 @@
+# This is the old job definition for upgrade tests.
+- job-group:
+    name: 'kubernetes-e2e-{provider}-{version-old}-{version-new}'
+    trigger-job: 'kubernetes-build'
+    test-owner: 'ihmccreery'
```

Comment:
nit: since you are here, can you fix the test-owner? For now, may be @davidopp can own it while he finds a new owner for them?